.github/workflows/tests.yaml

```diff
@@ -44,7 +44,7 @@ jobs:
   tests:
     strategy:
       matrix:
-        os: [macos-10.15, ubuntu-18.04, ubuntu-20.04, windows-2019]
+        os: [macos-12, ubuntu-18.04, ubuntu-20.04, windows-2019]
```

Comment:
is it on purpose that you don't use `macos-latest` instead?

Reply:
same reason for not using ubuntu-latest, we can discuss and make a general change if required.